### PR TITLE
[DKB] Add `fxRate` and `foreignCurrency` fields. Add support for price convertion. Ignore specific documents. Add support for different dividend format.

### LINKED ISSUE #556: DKB
## Documents
<details><summary>Verkauf_-_WKN_A14Q5E_-_Wertpapierabrechnung_vom_15.04.2020_zu_Depot_123456789_-_Ordernr._8647264300.pdf</summary><p>

```json
[
  [
    "Seite 1 von 2",
    "10919 Berlin",
    "Herrn",
    "Max Mustermann",
    "Musterstrasse 42",
    "12345 Musterstadt",
    "Depotnummer",
    "123456789",
    "Kundennummer",
    "123456789",
    "Max Mustermann",
    "Auftragsnummer",
    "864726/43.00",
    "Datum",
    "15.04.2020",
    "Rechnungsnummer",
    "W00883-0001653018/20",
    "Umsatzsteuer-ID",
    "DE137178746",
    "0883.04152317.0000227OR07",
    "Wertpapier Abrechnung Verkauf aus Kapitalmaßnahme",
    "Nominale",
    "Wertpapierbezeichnung",
    "ISIN",
    "(WKN)",
    "Stück 1.712",
    "DATA DEPOSIT BOX INC.",
    "REGISTERED SHARES O.N.",
    "CA2376321048",
    "(A14Q5E)",
    "Handels-/Ausführungsplatz",
    "Außerbörslich",
    "Abrech.-Preis",
    "0,0124916 CAD",
    "Wertpapierrechnung Lagerland Kanada",
    "Devisenkurs (EUR/CAD) 1,5268 vom 14.04.2020",
    "Kurswert",
    "14,01",
    "EUR",
    "Provision",
    "10,00-",
    "EUR",
    "Ermittlung steuerrelevante Erträge",
    "Veräußerungsverlust",
    "420,11-",
    "EUR",
    "Eingebuchte Aktienverluste",
    "420,11",
    "EUR",
    "Ausmachender Betrag",
    "4,01",
    "EUR",
    "Den Gegenwert buchen wir mit Valuta 16.04.2020 zu Gunsten des Kontos 1234567890",
    "(IBAN DE36 1234 0000 1234 1234 97), BLZ 12030000 (BIC BYLADEM1001).",
    "Die Wertpapiere entnehmen wir Ihrem Depotkonto.",
    "Steuerliche Ausgleichrechnung",
    "Verrechnete Aktienverluste",
    "235,67-",
    "Verrechneter Sparer-Pauschbetrag",
    "235,67",
    "Sofern keine Umsatzsteuer ausgewiesen ist, handelt es sich um eine umsatzsteuerbefreite Finanzdienstleistung.",
    "Barabfindung aus Fusion"
  ],
  [
    "Seite 2 von 2",
    "Depotnummer",
    "123456789",
    "Kundennummer",
    "123456789",
    "Auftragsnummer",
    "864726/43.00",
    "Datum",
    "15.04.2020",
    "0883.04152317.0000228OR07",
    "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
    "Nachrichtlich die Übersicht Ihrer Verrechnungs- und Steuertopfsalden zum Zeitpunkt der Erstellung der Abrechnung.",
    "Verlustverrechnungstöpfe 2020",
    "Berechnungsgrundlage",
    "der gezahlten Steuern",
    "Euro",
    "Aktien",
    "Sonstige",
    "Sparer-",
    "Pauschbetrag",
    "anrechenbare",
    "Quellensteuer",
    "Aktien und Sonstige",
    "Vorher",
    "0,00",
    "0,00",
    "565,19",
    "0,00",
    "0,00",
    "Verkauf",
    "420,11",
    "0,00",
    "0,00",
    "0,00",
    "0,00",
    "Steuerausgleich",
    "235,67-",
    "0,00",
    "235,67",
    "0,00",
    "0,00",
    "Nachher",
    "184,44",
    "0,00",
    "800,86",
    "0,00",
    "0,00",
    "Berücksichtigte Anschaffungsgeschäfte (alle ermittelten Beträge in EUR)",
    "Geschäft",
    "Auftragsnr.",
    "Ausführ.-tag",
    "Whr./St.",
    "Nennwert/Stück",
    "AS-Kosten",
    "Erlös",
    "ant. Ergebnis",
    "Kauf",
    "6523399100",
    "09.07.2015",
    "Stück",
    "550,0000",
    "211,71-",
    "1,29",
    "210,42-",
    "(D)",
    "Kauf",
    "6117106700",
    "02.05.2016",
    "Stück",
    "1.162,0000",
    "212,41-",
    "2,72",
    "209,69-",
    "(D)",
    "Summe aller Erträge nach Differenzmethode und/oder Ersatzbemessungsgrundlage",
    "420,11-"
  ]
]

```

</p></details>

<details><summary>Verkauf_-_WKN_A2JGMQ_-_Auftragsbestätigung_vom_09.07.2020_zu_Depot_123456789_-_Ordernr._4348056200.pdf</summary><p>

```json
[
  [
    "10919 Berlin",
    "Herrn",
    "Max Mustermann",
    "Musterstasse 123",
    "12345 Musterstadt",
    "Depotnummer",
    "123456789",
    "Kundennummer",
    "123456789",
    "Max Mustermann",
    "Auftragsnummer",
    "434805/62.00",
    "Datum",
    "09.07.2020",
    "0883.07092155.0011532OR03",
    "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
    "Wertpapier Auftragsbestätigung Verkauf",
    "Nominale",
    "Wertpapierbezeichnung",
    "ISIN",
    "(WKN)",
    "Stück 25",
    "ALFEN N.V.",
    "REGISTERED SHARES EO -,10",
    "NL0012817175",
    "(A2JGMQ)",
    "Handels-/Ausführungsplatz",
    "Tradegate (gemäß Weisung)",
    "Limit-Orde",
    "r",
    "Limit",
    "44,44 EUR",
    "Gültigkeit",
    "31.07.2020",
    "Auftraggeber",
    "Max Mustermann",
    "Auftragserteilung/ -ort",
    "Online-Banking",
    "Girosammelverw. mehrere Sammelurkunden - kein Stückeausdruck -",
    "Wir haben den Auftrag wie vorstehend angegeben in Vormerkung genommen. Zum Zeitpunkt der Erstellung dieser",
    "Bestätigung war die Order noch nicht ausgeführt.",
    "Für das Geschäft wurde keine Anlageberatung erbracht.",
    "Initiator der Besprechung war der Kunde."
  ]
]

```

</p></details>

<details><summary>Verkauf_-_WKN_A2JGMQ_-_Streichungsbestätigung_vom_24.07.2020_zu_Depot_123456789_-_Ordernr._4348056200.pdf</summary><p>

```json
[
  [
    "10919 Berlin",
    "Herrn",
    "Max Mustermann",
    "Musterstasse 123",
    "12345 Musterstadt",
    "Depotnummer",
    "123456789",
    "Kundennummer",
    "123456789",
    "Max Mustermann",
    "Auftragsnummer",
    "434805/62.00",
    "Datum",
    "24.07.2020",
    "0883.07242132.0011554OR05",
    "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
    "Wertpapier Streichungsbestätigung Verkauf",
    "Auftrag vom 09.07.2020 10:57:18 Uhr",
    "Nominale",
    "Wertpapierbezeichnung",
    "ISIN",
    "(WKN)",
    "Stück 25",
    "ALFEN N.V.",
    "REGISTERED SHARES EO -,10",
    "NL0012817175",
    "(A2JGMQ)",
    "Handels-/Ausführungsplatz",
    "Tradegate (gemäß Weisung)",
    "Limit-Orde",
    "r",
    "Limit",
    "44,44 EUR",
    "Gültigkeit",
    "31.07.2020",
    "Auftraggeber",
    "Max Mustermann",
    "Auftragserteilung/ -ort",
    "Online-Banking",
    "Girosammelverw. mehrere Sammelurkunden - kein Stückeausdruck -",
    "Vorstehenden Auftrag haben wir vorbehaltlich einer zwischenzeitlich erfolgten Orderausführung gestrichen.",
    "Für das Geschäft wurde keine Anlageberatung erbracht.",
    "Initiator der Besprechung war der Kunde.",
    "Die Order wurde auf Ihren Wunsch gestrichen."
  ]
]

```

</p></details>

<details><summary>Kauf_-_WKN_A2PWWB_-_Ausführungsanzeige_vom_26.08.2020_zu_Depot_123456789_-_Ordernr._4754928000.pdf</summary><p>

```json
[
  [
    "10919 Berlin",
    "Herrn",
    "Max Mustermann",
    "Musterstasse 123",
    "12345 Musterstadt",
    "Depotnummer",
    "123456789",
    "Kundennummer",
    "123456789",
    "Max Mustermann",
    "Auftragsnummer",
    "475492/80.00",
    "Datum",
    "26.08.2020",
    "0883.08262208.0011767OR02",
    "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
    "Wertpapier Ausführungsanzeige Kauf",
    "Nominale",
    "Wertpapierbezeichnung",
    "ISIN",
    "(WKN)",
    "Stück 55",
    "EHANG HOLDINGS LTD.",
    "REG. SHARES (SP.ADRS)1/2 O.N.",
    "US26853E1029",
    "(A2PWWB)",
    "Handels-/Ausführungsplatz",
    "NY Nasdaq (SM) (gemäß Weisung)",
    "Börsensegment",
    "XNMS",
    "Schlusstag/-Zeit",
    "26.08.2020 15:30:01",
    "Ortszeit",
    "26.08.2020 09:30:01",
    "Ausführungskurs",
    "8,89 USD",
    "Auftraggeber",
    "Max Mustermann",
    "Auftragserteilung/ -ort",
    "Online-Banking",
    "Wertpapierrechnung Lagerland USA",
    "Ihr Auftrag wurde zum oben genannten Kurs ausgeführt. Die rechtsverbindliche Abrechnung folgt.",
    "Für das Geschäft wurde keine Anlageberatung erbracht.",
    "Initiator der Besprechung war der Kunde."
  ]
]

```

</p></details>

<details><summary>Kauf_-_WKN_A2PWWB_-_Wertpapierabrechnung_vom_27.08.2020_zu_Depot_123456789_-_Ordernr._4754928000.pdf</summary><p>

```json
[
  [
    "10919 Berlin",
    "Herrn",
    "Max Mustermann",
    "Musterstasse 123",
    "12345 Musterstadt",
    "Depotnummer",
    "123456789",
    "Kundennummer",
    "123456789",
    "Max Mustermann",
    "Auftragsnummer",
    "475492/80.00",
    "Datum",
    "27.08.2020",
    "Rechnungsnummer",
    "W00883-0003940664/20",
    "Umsatzsteuer-ID",
    "DE137178746",
    "0883.08272145.0011114OR07",
    "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
    "Wertpapier Abrechnung Kauf",
    "Auftrag vom 26.08.2020 14:10:30 Uhr",
    "Nominale",
    "Wertpapierbezeichnung",
    "ISIN",
    "(WKN)",
    "Stück 55",
    "EHANG HOLDINGS LTD.",
    "REG. SHARES (SP.ADRS)1/2 O.N.",
    "US26853E1029",
    "(A2PWWB)",
    "Handels-/Ausführungsplatz",
    "NY Nasdaq (SM) (gemäß Weisung)",
    "Börsensegment",
    "XNMS",
    "Market-Orde",
    "r",
    "Limit",
    "billigst",
    "Schlusstag/-Zeit",
    "26.08.2020 15:30:01",
    "Ortszeit",
    "26.08.2020 09:30:01",
    "Ausführungskurs",
    "8,89 USD",
    "Auftraggeber",
    "Max Mustermann",
    "Auftragserteilung/ -ort",
    "Online-Banking",
    "Wertpapierrechnung Lagerland USA",
    "Devisenkurs (EUR/USD) 1,1761 vom 27.08.2020",
    "Kurswert",
    "415,74-",
    "EUR",
    "Provision",
    "20,00-",
    "EUR",
    "Fremde Auslagen",
    "5,95-",
    "EUR",
    "Ausmachender Betrag",
    "441,69-",
    "EUR",
    "Den Gegenwert buchen wir mit Valuta 28.08.2020 zu Lasten des Kontos 1234567890",
    "(IBAN DE36 1234 0000 1234 1234 97), BLZ 12030000 (BIC BYLADEM1001).",
    "Die Wertpapiere schreiben wir Ihrem Depotkonto gut.",
    "Sofern keine Umsatzsteuer ausgewiesen ist, handelt es sich um eine umsatzsteuerbefreite Finanzdienstleistung.",
    "Für das Geschäft wurde keine Anlageberatung erbracht.",
    "Initiator der Besprechung war der Kunde."
  ]
]

```

</p></details>

<details><summary>Kauf_-_WKN_A115DQ_-_Auftragsbestätigung_vom_19.12.2017_zu_Depot_123456789_-_Ordernr._4032685900.pdf</summary><p>

```json
[
  [
    "10919 Berlin",
    "Herrn",
    "Max Mustermann",
    "Musterstrasse 42",
    "12345 Musterstadt",
    "Depotnummer",
    "123456789",
    "Kundennummer",
    "123456789",
    "Max Mustermann",
    "Auftragsnummer",
    "403268/59.00",
    "Datum",
    "19.12.2017",
    "0883.12192120.0004664OR03",
    "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
    "Wertpapier Auftragsbestätigung Kauf",
    "Nominale",
    "Wertpapierbezeichnung",
    "ISIN",
    "(WKN)",
    "Stück 1.041",
    "DIGITALX LTD.",
    "REGISTERED SHARES O.N.",
    "AU000000DCC9",
    "(A115DQ)",
    "Handels-/Ausführungsplatz",
    "Tradegate (Best Execution)",
    "Limit-Orde",
    "r",
    "Limit",
    "0,23 EUR",
    "Gültigkeit",
    "19.12.2017",
    "Auftraggeber",
    "Max Mustermann",
    "Auftragserteilung/ -ort",
    "Online-Banking",
    "Wertpapierrechnung Lagerland Australien",
    "Wir haben den Auftrag wie vorstehend angegeben in Vormerkung genommen. Zum Zeitpunkt der Erstellung dieser",
    "Bestätigung war die Order noch nicht ausgeführt.",
    "Für das Geschäft wurde keine Anlageberatung erbracht.",
    "Initiator der Besprechung war der Kunde."
  ]
]

```

</p></details>

<details><summary>Kauf_-_WKN_A115DQ_-_Streichungsbestätigung_vom_20.12.2017_zu_Depot_123456789_-_Ordernr._4032685900.pdf</summary><p>

```json
[
  [
    "10919 Berlin",
    "Herrn",
    "Max Mustermann",
    "Musterstrasse 42",
    "12345 Musterstadt",
    "Depotnummer",
    "123456789",
    "Kundennummer",
    "123456789",
    "Max Mustermann",
    "Auftragsnummer",
    "403268/59.00",
    "Datum",
    "20.12.2017",
    "0883.12202105.0003295OR05",
    "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
    "Wertpapier Streichungsbestätigung Kauf",
    "Auftrag vom 19.12.2017 07:47:16 Uhr",
    "Nominale",
    "Wertpapierbezeichnung",
    "ISIN",
    "(WKN)",
    "Stück 1.041",
    "DIGITALX LTD.",
    "REGISTERED SHARES O.N.",
    "AU000000DCC9",
    "(A115DQ)",
    "Handels-/Ausführungsplatz",
    "Tradegate (Best Execution)",
    "Limit-Orde",
    "r",
    "Limit",
    "0,23 EUR",
    "Gültigkeit",
    "19.12.2017",
    "Auftraggeber",
    "Max Mustermann",
    "Auftragserteilung/ -ort",
    "Online-Banking",
    "Wertpapierrechnung Lagerland Australien",
    "Vorstehenden Auftrag haben wir vorbehaltlich einer zwischenzeitlich erfolgten Orderausführung gestrichen.",
    "Für das Geschäft wurde keine Anlageberatung erbracht.",
    "Initiator der Besprechung war der Kunde.",
    "Die Order wurde wegen Fristablauf gelöscht."
  ]
]

```

</p></details>

---
Issue submitted with https://import.tresor.one

### LINKED ISSUE #557: DKB
## Document
<details><summary>Verkauf_-_WKN_A14Q5E_-_Wertpapierabrechnung_vom_15.04.2020_zu_Depot_501107049_-_Ordernr._8647264300.pdf</summary><p>

```json
[
    [
      "Seite 1 von 2",
      "10919 Berlin",
      "Herrn",
      "Max Mustermann",
      "Musterstrasse 42",
      "12345 Stadt",
      "Depotnummer",
      "123456789",
      "Kundennummer",
      "123456789",
      "Max Mustermann",
      "Auftragsnummer",
      "123456/43.00",
      "Datum",
      "15.04.2020",
      "Rechnungsnummer",
      "W00883-0001653018/20",
      "Umsatzsteuer-ID",
      "DE137178746",
      "0883.04152317.0000227OR07",
      "Wertpapier Abrechnung Verkauf aus Kapitalmaßnahme",
      "Nominale",
      "Wertpapierbezeichnung",
      "ISIN",
      "(WKN)",
      "Stück 1.712",
      "DATA DEPOSIT BOX INC.",
      "REGISTERED SHARES O.N.",
      "CA2376321048",
      "(A14Q5E)",
      "Handels-/Ausführungsplatz",
      "Außerbörslich",
      "Abrech.-Preis",
      "0,0124916 CAD",
      "Wertpapierrechnung Lagerland Kanada",
      "Devisenkurs (EUR/CAD) 1,5268 vom 14.04.2020",
      "Kurswert",
      "14,01",
      "EUR",
      "Provision",
      "10,00-",
      "EUR",
      "Ermittlung steuerrelevante Erträge",
      "Veräußerungsverlust",
      "420,11-",
      "EUR",
      "Eingebuchte Aktienverluste",
      "420,11",
      "EUR",
      "Ausmachender Betrag",
      "4,01",
      "EUR",
      "Den Gegenwert buchen wir mit Valuta 16.04.2020 zu Gunsten des Kontos 123456789",
      "(IBAN DE36 1234 0000 1234 1234 12), BLZ 12030000 (BIC BYLADEM1001).",
      "Die Wertpapiere entnehmen wir Ihrem Depotkonto.",
      "Steuerliche Ausgleichrechnung",
      "Verrechnete Aktienverluste",
      "235,67-",
      "Verrechneter Sparer-Pauschbetrag",
      "235,67",
      "Sofern keine Umsatzsteuer ausgewiesen ist, handelt es sich um eine umsatzsteuerbefreite Finanzdienstleistung.",
      "Barabfindung aus Fusion"
    ],
    [
      "Seite 2 von 2",
      "Depotnummer",
      "123456789",
      "Kundennummer",
      "123456789",
      "Auftragsnummer",
      "123456/43.00",
      "Datum",
      "15.04.2020",
      "0883.04152317.0000228OR07",
      "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
      "Nachrichtlich die Übersicht Ihrer Verrechnungs- und Steuertopfsalden zum Zeitpunkt der Erstellung der Abrechnung.",
      "Verlustverrechnungstöpfe 2020",
      "Berechnungsgrundlage",
      "der gezahlten Steuern",
      "Euro",
      "Aktien",
      "Sonstige",
      "Sparer-",
      "Pauschbetrag",
      "anrechenbare",
      "Quellensteuer",
      "Aktien und Sonstige",
      "Vorher",
      "0,00",
      "0,00",
      "565,19",
      "0,00",
      "0,00",
      "Verkauf",
      "420,11",
      "0,00",
      "0,00",
      "0,00",
      "0,00",
      "Steuerausgleich",
      "235,67-",
      "0,00",
      "235,67",
      "0,00",
      "0,00",
      "Nachher",
      "184,44",
      "0,00",
      "800,86",
      "0,00",
      "0,00",
      "Berücksichtigte Anschaffungsgeschäfte (alle ermittelten Beträge in EUR)",
      "Geschäft",
      "Auftragsnr.",
      "Ausführ.-tag",
      "Whr./St.",
      "Nennwert/Stück",
      "AS-Kosten",
      "Erlös",
      "ant. Ergebnis",
      "Kauf",
      "6523399100",
      "09.07.2015",
      "Stück",
      "550,0000",
      "211,71-",
      "1,29",
      "210,42-",
      "(D)",
      "Kauf",
      "6117106700",
      "02.05.2016",
      "Stück",
      "1.162,0000",
      "212,41-",
      "2,72",
      "209,69-",
      "(D)",
      "Summe aller Erträge nach Differenzmethode und/oder Ersatzbemessungsgrundlage",
      "420,11-"
    ]
  ]

```

</p></details>

---
Issue submitted with https://import.tresor.one

### LINKED ISSUE #617: Broker DKB; keine Einlesung von Dividendenzahlung Telekom
## Document
<details><summary>WKN_555750_-_Ertragsabrechnung_Dividenden_vom_23.06.2020_zu_Depot_501691364_-_Belegnr._67511687440.pdf</summary><p>

```json
[
  [
    "10919 Berlin",
    "Herrn",
    "Yrp Eakrkcon",
    "Uqdugtsyrj. 4",
    "61781 Xyamzbbbk",
    "Depotnummer",
    "416184358",
    "Kundennummer",
    "4498371741",
    "Xkz Yjsactty",
    "Abrechnungsnr.",
    "21509164308",
    "Datum",
    "23.06.2020",
    "0883.06240101.0010940ER01",
    "Dieses Dokument wurde maschinell erstellt und wird nicht unterschrieben.",
    "Bitte ggf. Rückseite beachten.",
    "Ertragsgutschrift nach § 27 KStG",
    "Nominale",
    "Wertpapierbezeichnung",
    "ISIN",
    "(WKN)",
    "Stück 32",
    "DEUTSCHE TELEKOM AG",
    "NAMENS-AKTIEN O.N.",
    "DE0005557508",
    "(555750)",
    "Zahlbarkeitstag",
    "24.06.2020",
    "Bestandsstichtag",
    "19.06.2020",
    "Ex-Tag",
    "22.06.2020",
    "Geschäftsjahr",
    "01.01.2019 - 31.12.2019",
    "Ertrag",
    "pro Stück",
    "0,60",
    "EUR",
    "Dividendengutschrift nach § 27 KStG",
    "19,20+",
    "EUR",
    "Ausmachender Betrag",
    "19,20+",
    "EUR",
    "Lagerstelle Clearstream Banking FFM (849000 / 40030000)",
    "Den Betrag buchen wir mit Wertstellung 24.06.2020 zu Gunsten des Kontos 1037219654 (IBAN UJ46 9181 6559 3097",
    "2196 54), BLZ 120 300 00 (BIC BYLADEM1001).",
    "Keine Steuerbescheinigung.",
    "Es handelt sich bei der Ausschüttung um Leistungen aus dem steuerlichen Einlagekonto der Kapitalgesellschaft (§27 Abs.",
    "1-7 KStG). Dieser Ertrag unterliegt laut Gesetz zum Zeitpunkt des Zuflusses keinem Steuerabzug und ist",
    "einkommensteuerfrei. Er mindert jedoch im Nachhinein den Kaufkurs der bezogenen Aktie, so dass bei deren Verkauf",
    "möglicherweise ein entsprechend höherer Kursgewinn zu versteuern ist."
  ],
  []
]

```

</p></details>

---
Issue submitted with https://import.tresor.one

### PR DESCRIPTION
**Pull request by dredav**
- Fix #556
- Fix #557
- Fix #617
- Add some samples from #556, #557 and #617 as new tests cases
- Some documents don't have a order execution date. We need to search some other possible date values
- Resolve issue where the foreign currency amount was used as price without convertion
- Made the `findPayout` logic better with an other dividend format.
- Ignore some documents with status code `7`

### CODE REVIEW

**Comment by SirGibihm** [comment]
Some merge conflicts arose due to the mass refactor of `canParseFirstPage` to `canParseDocument`

File: src/brokers/dkb.js [hunk 1]
```diff
@@ -30,8 +30,34 @@ const findCompany = (textArr, pieceIdx, isinIdx) =>
...
+    const regex = /(\d{2}\.\d{2}\.\d{4})/;
+    if (lineIndex > 0 && regex.test(content[lineIndex])) {
+      dateLine = regex.exec(content[lineIndex])[0];
+    }
+  }
+
+  if (dateLine === '') {
+    // Last one: Get the date of the document. This is not the real order date but it's better than nothing.
+    dateLine = getValueByPreviousElement(content, 'Datum');
+  }
```
Comment (SirGibihm) [inline src/brokers/dkb.js:57]: Does this happen somewhere in your tests?

File: src/brokers/dkb.js [hunk 2]
```diff
@@ -145,6 +198,38 @@ const findTax = pages => {
   return +totalTax;
 };
 
+const findForeignInformation = content => {
+  let fxRate, foreignCurrency, baseCurrency;
```
Comment (SirGibihm) [inline src/brokers/dkb.js:202]: a small comment to explain what is returned in which format would be nice.

**Review by SirGibihm — CHANGES_REQUESTED** [review]
If you could add this comment I will merge

File: src/brokers/dkb.js (hunk 1 above)
Comment (dredav) [reply src/brokers/dkb.js:57]: No, but I want to offer a fallback solution for the worse case. What do you think?

File: src/brokers/dkb.js (hunk 2 above)
Comment (dredav) [reply src/brokers/dkb.js:202]: Sure. I use the same comment from `flatex.findForeignInformation`. Done!

**Comment by dredav** [comment]
Conflicts solved.

**Review by SirGibihm — APPROVED** [review]

**Pull request merged**